leshan-standalone/src/main/resources/webapp/js/client-controllers.js

```diff
@@ -52,6 +52,28 @@ lwClientControllers.controller('ClientListCtrl', [
             $location.path('/clients/' + client.endpoint);
         };
 
+        // the tooltip message to display for a client (all standard attributes, plus additional ones)
+        $scope.clientTooltip = function(client) {
+            var standard = ["Lifetime : " + client.lifetime + "s",
+                            "Binding mode : " + client.bindingMode];
+
+            if (client.lwm2mVersion) {
```

Comment:
The lwm2m version has a default value, so this field must not be null.
In practice, it is null because there is a typo [here](https://github.com/eclipse/leshan/pull/67/files#diff-8a0140268bfea09bcd262f537197a990R45) (too many MmMmM :p)